### PR TITLE
Move tributejs to npm/webpack

### PR DESCRIPTION
**Pull request by silverwind**
- Move vendored bundle to npm and webpack
- Rewrote initialization to single function
- Restyled it (made it a bit smaller)
- Fixed it for arc-green

image

image

image

### CODE REVIEW

File: web_src/less/_tribute.less [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
+.tribute-container ul {
+    margin-top: 0 !important;
+    background: #ffffff !important;
+}
+
+.tribute-container li {
+    padding: 3px .5rem !important;
+}
+
+.tribute-container img span.fullname {
```
Comment (mrsdizzie) [inline web_src/less/_tribute.less]:
```suggestion
.tribute-container li span.fullname {
```

**Comment by silverwind** [comment]
Note the warnings about tribute being bound twice are still present. I'm not sure yet how to fix but it's no blocker as the warning is present on master too.

**Review by CirnoT — APPROVED** [review]

**Comment by silverwind** [comment]
Will have a look later into those warnings, I think the initialization can be combined into one `Tribute` instance to fix it.

**Comment by silverwind** [comment]
Double initialization is fixed for the case of a single `attachTribute` call and the warning should be gone when opening a issue. It's still possible to occur if `attachTribute` is called twice on the same element, but I don't think we do that currently.

**Comment by silverwind** [comment]
I wonder why @CirnoT's review did not count.

**Review by gary-kim — APPROVED** [review]
LGTM

**Comment by techknowlogick** [comment]
ping LG-TM

**Pull request merged**